scripts/apps/authoring/authoring/directives/ItemAssociationDirective.ts

```diff
@@ -66,6 +66,15 @@ export function ItemAssociationDirective(renditions, notify) {
                     // which it shouldn't do, since the element is not controlled by the library.
                     // Because of this, drop is triggered when it shouldn't have been on firefox, but not on chrome.
                     // The same media types check is added from `dragover` method to work around the situation.
+                    const externalItemsCount = Object.values(event.originalEvent.dataTransfer.files || []).length;
+
+                    if (externalItemsCount > 1) {
```

Comment:
The comment block above refers to the `if` statement that was directly below it. You have now added code between the code and its comment which makes the comment look like it's about your newly added code, which is not the case. Move the comment block down so its directly above this line `if (isAllowedMediaType(scope, event)) {`.

Also, add a comment on what your code is doing and what use case does it handle. Why wouldn't you want to allow adding multiple files?

Reply:
I feature-media field we support only one media, for more than one media fields we have custom-media fields.